lib/challenge.js

```diff
@@ -48,7 +62,14 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
+
+name: ,
+species: ,
+breed: ,
+noise: ,
+
+ ;}
```

Comment:
This is a mostly-good framework for an object literal. The colons are where they should be, and the commas are where they should be, save the last one -- the last key-value pair in an object literal should not be followed with a comma, no more than the last word in a sentence should be followed with a space.

The biggest problem here is that semicolon. Semicolons are not allowed characters in object literal syntax. They always lead to a syntax error. It should come after the closing curly brace.

A less serious problem is the absence of values for these four attributes. This is why I said it's a mostly-good framework; it's a good place to be before deciding which values to assign to each attribute. Scaffolding object literals like this can be very helpful.